### ISSUE TITLE
[Request]: Different colors in player database

### ISSUE THREAD

**Issue by ghost** [YimMenu/YimMenu#1396]
### Problem

Some players appear in the table as green, but they seem to be on a mission or somewhere else and there is no way to connect to them and the message "session no longer exists" is displayed

### Solution

Maybe it would be worth giving them a yellow color? For example, for those who are on a mission but cannot be visited. Or is it impossible to determine whether he is in the mission or not?

### Reason

For ease of viewing

**Comment by binded-zz** [comment]
i dont think that any one will add colors i tried to already and every one said no its not good 
it says session no longer exists   is because it only shows that there are online  not so much that there in a game 
idk know if theres a way to add if there ingame 

idk your just gone to have to figure it out your self or just wait till they get around to adding some thing like that
